vuepress/docs/next/tutorials/ecr/github-actions-workflow-from-ecg.md

```diff
@@ -0,0 +1,219 @@
+
+The final Job definition looks like this:
+``` yaml
+micro-frontends:
+ name: micro frontend job
+ runs-on: ubuntu-latest
+ steps:
+   - uses: actions/checkout@v2
+   - uses: actions/setup-node@v2.1.4
+     with:
+       node-version: '14.15.0'
+   - name: Run tests
+     run: |
+       cd <mfe_path>
+       npm install
+       npm test
+```
+
+### Use a matrix strategy for a Job with multiple micro frontends
+
+If you have multiple micro frontends located in the source project, you can use a matrix strategy to define multiple Jobs and avoid repeating definitions. 
+
+Building on the previous example:
+1. Insert a `strategy` definition after the `runs-on` property. Please note we are disabling the fail-fast option to allow all the Jobs to run, even if one fails.
+``` yaml
+strategy:
+ fail-fast: false
+```
+2. Because all the micro frontends are located in the same root folder, you can define your matrix with the MFE names and build paths
+``` yaml
+matrix:
+ mfe:
+   - my-mfe-1
+   - my-mfe-2
+```
+
+The final Job should look like this:
+``` yaml
+micro-frontends:
+ name: ${{ matrix.mfe }} micro frontend
+ runs-on: ubuntu-latest
+ strategy:
+   fail-fast: false
+   matrix:
+     mfe:
+       - my-mfe-1
+       - my-mfe-2
+ steps:
+   - uses: actions/checkout@v2
+   - uses: actions/setup-node@v2.1.4
+     with:
+       node-version: '14.15.0'
+   - name: Run tests
+     run: |
+       cd $GITHUB_WORKSPACE/$PROJECT_NAME/my_base_path/${{ matrix.mfe }}
+       npm install
+       npm run test
+```
+
+Below is the complete Workflow file:
+
+``` yaml
+name: Application CI
+on: push
+env:
+ PROJECT_NAME: my-project
+jobs:
+ backend:
+   runs-on: ubuntu-latest
+   steps:
+     - uses: actions/checkout@v2
+     - uses: actions/setup-java@v1
+       with:
+         java-version: '11.x'
+     - name: Stop MySQL server
+       run: sudo /etc/init.d/mysql stop
+     - name: Run backend test
+       run: |
+         cd $GITHUB_WORKSPACE/$PROJECT_NAME
+         chmod +x mvnw
+         ./mvnw -ntp clean test
+     - name: Package application
+       run: |
+         cd $GITHUB_WORKSPACE/$PROJECT_NAME
+         ./mvnw -ntp package -Pprod -DskipTests
+ micro-frontends:
+   name: ${{ matrix.mfe }} micro frontend
+   runs-on: ubuntu-latest
+   strategy:
+     fail-fast: false
+     matrix:
+       mfe:
+         - my-mfe-1
+         - my-mfe-2
+   steps:
+     - uses: actions/checkout@v2
+     - uses: actions/setup-node@v2.1.4
+       with:
+         node-version: '14.15.0'
+     - name: Run tests
+       run: |
+         cd $GITHUB_WORKSPACE/$PROJECT_NAME/my_base_path/${{ matrix.mfe }}
+         npm install
+         npm run test
+```
+
+
+### Run the Workflow
+To run and test this Workflow, commit and push the definition file to a branch in your repository. The Workflow will start according to the events defined for the `on` property (if you have available runners). Completed and in-progress Workflows can be accessed under the "Actions" tab in your repository.
```

Comment:
think the parenthesis is unnecesssary here but minor.

Reply:
i think it may be a subtle point that gets attention and clarification with the parentheses... i can see it either way